.jcheck/conf

```diff
@@ -36,4 +36,4 @@ domain=openjdk.org
 files=.*\.java$|.*\.c$|.*\.h$|.*\.cpp$|.*\.hpp$
 
 [checks "reviewers"]
-reviewers=1
+committers=1
```

Comment:
Same here.

Reply:
Interesting. The conf file wasn't even in the list of conflicting files.